.drone.yml

```diff
@@ -7,3 +7,7 @@ steps:
   settings:
     repo: docker.funzt-halt.net/mqtt-pushgateway
     tags: latest
+
+trigger:
+  branch:
+  - master
```

Comment:
I applied a similar change to .drone.yml, please undo these - thanks for the reminder though :man_facepalming: